mmengine/visualization/visualizer.py

```diff
@@ -412,26 +441,66 @@ def draw_points(self,
         assert positions.shape[-1] == 2, (
             'The shape of `positions` should be (N, 2), '
             f'but got {positions.shape}')
-        colors = color_val_matplotlib(colors)  # type: ignore
-        self.ax_save.scatter(
-            positions[:, 0], positions[:, 1], c=colors, s=sizes, marker=marker)
+        if self.backend == 'matplotlib':
+            colors = color_val_matplotlib(colors)  # type: ignore
+            self.ax_save.scatter(
+                positions[:, 0],
+                positions[:, 1],
+                c=colors,
+                s=sizes,
+                marker=marker)
+        else:
+            positions = positions.tolist()
+            if isinstance(colors, str) or isinstance(colors, tuple):
+                colors = [colors] * len(positions)
+            colors = color_val_opencv(colors)  # type:ignore
+            if sizes is not None:
+                sizes = tensor2ndarray(sizes)
+            if marker is None:
+                for i, pos in enumerate(positions):
+                    pos = (int(pos[0]), int(pos[1]))
+                    cv2.circle(
+                        img=self._image,
+                        center=pos,
+                        radius=int(sizes[i]) if sizes is not None else 6,
+                        color=colors[i],
+                        thickness=-1)
+            else:
+                if not isinstance(marker, int):
+                    raise TypeError(
+                        'The type of `marker` in `cv2` should be int, '
+                        f'but got {type(marker)}')
+                if sizes is None:
+                    sizes = [self._default_cv2_markersize] * len(positions)
+                for pos, color, size in zip(positions, colors, sizes):
+                    kwargs = {
+                        'img': self._image,
+                        'position': pos,
+                        'color': color,
+                        'markerType': marker,
+                        'markerSize': int(size)
+                    }
+                    cv2.drawMarker(**kwargs)
+
         return self
 
     @master_only
     def draw_texts(
         self,
         texts: Union[str, List[str]],
         positions: Union[np.ndarray, torch.Tensor],
-        font_sizes: Optional[Union[int, List[int]]] = None,
-        colors: Union[str, tuple, List[str], List[tuple]] = 'g',
+        font_sizes: Optional[Union[int, float, List[int], List[float]]] = None,
+        colors: Union[str, tuple, List[Union[str, tuple]]] = 'g',
         vertical_alignments: Union[str, List[str]] = 'top',
         horizontal_alignments: Union[str, List[str]] = 'left',
         font_families: Union[str, List[str]] = 'sans-serif',
+        font_faces: Union[int, List[int]] = cv2.FONT_HERSHEY_SIMPLEX,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.